README.md

```diff
@@ -1,12 +1,12 @@
-OpenShift Application Platform
+Origin Application Platform
 ==============================
 
 [![GoDoc](https://godoc.org/github.com/openshift/origin?status.png)](https://godoc.org/github.com/openshift/origin)
 [![Travis](https://travis-ci.org/openshift/origin.svg?branch=master)](https://travis-ci.org/openshift/origin)
 
-This is the source repository for [OpenShift 3](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
+This is the source repository for [Origin](https://openshift.github.io), based on top of [Docker](https://www.docker.io) containers and the
```

Comment:
Can't change - but you can add a separate sentence. "Origin is a distribution of Kubernetes optimized for enterprise application development and deployment, used by OpenShift and Atomic Enterprise".